# Evidence Summary Template v5.1

---

### Version 5.1: January 27, 2025

ClinGen Gene Curation Standardized Evidence Summary Text
Instructions: Evidence summaries are an important part of the gene-disease validity curation and are required as of SOP version 8. These evidence summaries are made publicly available on our website, www.clinicalgenome.org, and can be accessed by different types of users, including those who may not be readily familiar with a given gene-disease relationship.
As such, evidence summaries should include an overview of the evidence used to make the classification, as well as any other information necessary to understand the relationship between the gene and the disease. The evidence summary is not meant to be exhaustive but should be clear and understandable to a wide audience.
The following are suggestions for elements to include in an evidence summary, and how they might be worded. Choose the text that is relevant for the gene curation at hand (not all text will be relevant for every curation). Bracketed text acts as a placeholder and should be replaced with specific information. Note that there is no specified order in which these elements should appear; please feel free to mix and match the appropriate elements, and to edit the suggested text for flow, clarity, etc. as needed.
Table of contents
- General Description of Gene/Disease
- General Description of Relevant Evidence
- Other relevant information
- Summary Statement:
- Templates:
○ Combined template example (for a Definitive classification)
○ General Description Template
○ Lumping and Splitting Template
○ Summary Statement template
○ Experimental evidence template

## - Recuration


## - Appendix

General Description of Gene/Disease
- Include the gene name, disease, and mode of inheritance.
- Note the year of the first clinical report and provide a reference.
- Consider providing a brief synopsis of the gene and its function.
- Consider providing a brief synopsis of the clinical features of the disease, particularly if this information is not readily conveyed in the disease name (e.g., eponymous disease names, disease names referencing the gene name, etc.).
- If a precuration was performed and disease entities were either lumped or split, the evidence summary should include a brief description of why this decision was made,
which entities are included in the current curation, and which entities will be addressed in



separate curations, if applicable. See the Lumping and Splitting guidelines for points to consider when deciding to lump or split.
- Include information about how many unique variants have been reported to support this gene-disease relationship (“At least XX unique variants have been reported” or “Over XX
unique variants have been reported,” etc.) across how many publications. If there are too many to count, this can be general (e.g. “multiple,” “numerous,” etc.).
- Include information about the variant spectrum/what types of variants have been observed (e.g. missense, in-frame indel, nonsense, frameshift, large deletion, complex rearrangement, etc.).
General Description of Relevant Evidence
- Describe the type of evidence supporting or refuting the gene-disease relationship
(case-level data, case-control data, segregation data, experimental data). Provide a basic description of each.
- Include the mechanism of disease, if known. If the mechanism is unknown or unclear,
this may be stated as well.
- If applicable, note any important founder variants, recurrent variants, etc.
- Note if the maximum score for a particular evidence category was reached, but more information is available in the literature.
- When describing case-control studies, include the total number of case-control studies evaluated (including PMIDs), whether or not they were single-variant studies or aggregate variant studies, information about the reported significance and/or odds ratio,
and any other relevant information about power, bias, or confounding necessary to describe the quality of the study.
Other relevant information
Include any other information the GCEP feels is important for someone seeking information on this gene/disease relationship to understand. This may include (but is not limited to) the following:
- Evidence not able to be formally entered into the GCI (for example, preprint articles with
DOIs instead of PMIDs, etc.)..In the interim, please document any relevant information in the evidence summary.
- Any variants evaluated but not scored, particularly if the final classification is disputed or refuted. Though variant-level scoring decisions are displayed for the public on the website, it is still helpful to provide a discussion regarding why a disputed or refuted classification was reached. If a gene/disease relationship is classified as No known disease relationship, specify why it was evaluated by the GCEP. This may include it if it was present on clinical testing panels, etc.
- Phenotypic variability, including reduced penetrance, variable expressivity, known intrafamilial variability, different phenotypes associated with particular variants etc.
- Known genotype-phenotype correlations



- Known treatments (e.g. approved gene therapies)
- Etc.
### Summary Statement:

- Summarize the final clinical validity of the gene-disease pair.
- Include the date the curation was approved and by which GCEP.
- If the entry is a recuration:
○ State when the original curation was performed, what the original classification was, and which GCEP performed the original evaluation (if different from the current GCEP)
○ Explain if the classification has or has not changed and the evidence supporting this change.



Templates:A combination of any of the following templates can be used based on the final gene-disease clinical validity classification and evidence assessed. These templates are provided as a guide, however a gene curation expert panel can choose to draft their own templates, but should include the relevant points described above. Bracketed text acts as a placeholder and should be replaced with specific information.
### Combined template example (for a Definitive classification):

The following represents an example evidence summary for a Definitive classification. Each section of text represents a different section of the evidence summary, and is described in more detail below. Not all elements included in this example are relevant to each curation; use this example as a general guide, but edit as appropriate. Additional template text and published examples are included at the bottom of this document.
[Gene] was first reported in relation to [mode of inheritance] [disease] in [year] (Author et al., PMID: XXX). [Optional brief synopsis of the disease]. Per criteria outlined by the ClinGen
Lumping and Splitting guidelines, we found [no difference/difference] in [molecular mechanism(s) AND/OR inheritance pattern AND/OR phenotypic variability]. Therefore, the following disease entities have been [lumped/split] into [one/multiple] disease entities,
[disease1] (OMIM:XXX), [disease2] (OMIM:XXX)... [more L&S evidence if needed]
[previous sentences only applicable if there are differences and/or multiple disease entities]. The split curation for [mode of inheritance] [disease] has been curated separately
(if applicable). [Number] variants (variant type, e.g. missense, in-frame indel, nonsense,
frameshift, large deletion, complex rearrangement, etc.) that have been reported in [Number]
probands in [Number] publications (PMIDs: XXX, XXX) are included in this curation.
[Additional essential genetic evidence description]. More evidence is available in the literature, but the maximum score for genetic evidence (12 pts.) has been reached (if applicable). The mechanism of pathogenicity [appears/is reported/is known] to be [XXX] (e.g.
loss of function, gain of function, dominant negative, unknown, unclear). This gene-disease relationship is also supported by [experimental evidence] (e.g. animal models, expression studies, in vitro functional assays, etc.) (PMIDs: XXX, XXX). [essential experimental evidence description]. [Add other gene-specific relevant information here.] In summary, there is definitive evidence supporting the relationship between [Gene] and [mode of inheritance]
[disease]. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time (need to change to other wording if not definitive,
according to the ClinGen template).This classification was approved by the ClinGen [NAME
OF GENE CURATION EXPERT PANEL] GCEP on the meeting date [Name of Month, Date,
Year] (SOP Version [#])].



### General description template:

[GENE ID] was FIRST reported in relation to [mode of inheritance] [disease] in [year]
(Author et al., PMID). [Optional brief synopsis of the disease]. At least [# UNIQUE
VARIANTS] variants (e.g. missense, in-frame indel, nonsense, frameshift, large deletion,
complex rearrangement, etc.) have been reported in humans. Evidence supporting/refuting
[edit as appropriate] this gene-disease relationship includes DATA-TYPE (e.g. case-level data, case-control data, segregation data, experimental data [choose all that apply]).
### Lumping and splitting template:

Variants in [Gene ID] have been reported in individuals with the following disease entities: [list entities and/or phenotypes as found in OMIM, Orphanet, or the literature]. Per criteria outlined by the ClinGen Lumping and Splitting guidelines, we found [no difference/difference]
in [molecular mechanism(s) AND/OR inheritance pattern AND/OR phenotypic variability].
Therefore, the following disease entities have been [lumped/Split] into [one/multiple] disease entities, [disease1] (OMIM:XXX), [disease2] (OMIM:XXX)... [more L&S evidence if needed].
The split curation for [mode of inheritance] [disease] has been curated separately (if applicable).
- Additional Lumping and splitting options: Of note, this gene has also been implicated in
(other disease assertions). These will be/have been assessed separately.
### Experimental evidence template:

This gene-disease relationship is supported by (relevant experimental evidence used e.g.
animal models, expression studies, in vitro functional assays, etc.; PMIDs) [essential experimental evidence description]
### Summary statement template:

### Summary of Case Level Data:

Variants in this gene have been reported in at least # or % probands in # publications (PMIDs).
Variants in this gene segregated with disease in # additional family members.
### Summary of Case-Control Data:

This gene-disease relationship has been studied in at least XXX case-control studies (PMIDs)
at the single variant level and XXX case-control studies at the aggregate variant level.
Comment on overall statistics OR.
### For STRONG/DEFINITIVE genes (if applicable):

More evidence is available in the literature, but the maximum score for genetic evidence and/or experimental evidence (12 pts.) has been reached.



The mechanism for disease is (unknown) OR (homozygous loss of function/
haploinsufficiency/ heterozygous gain of function, unclear, etc.) (Reference, if available)
Include one of the following summary statements that correspond to the final gene-disease validity classification:
- DEFINITIVE: In summary, there is definitive evidence to support the relationship between [GENE] and [INHERITANCE & DISEASE]. This has been repeatedly demonstrated in both the research and clinical diagnostic settings and has been upheld over time.
- STRONG: In summary, there is strong evidence to support the relationship between
[GENE] and [INHERITANCE & DISEASE]. Three years must elapse from the first proposal of the assertion to reach a definitive classification without any valid contradictory evidence. We will re-evaluate this gene-disease relationship at that time to determine if an upgraded classification of definitive is warranted.
- MODERATE: In summary, there is moderate evidence to support this gene-disease relationship. While more evidence is needed to establish this relationship definitively, no convincing contradictory evidence has emerged.
- LIMITED: In summary, there is limited evidence to support this gene-disease relationship. Although more evidence is needed to support a causal role, no convincing evidence has emerged that contradicts the gene-disease relationship.
- NO KNOWN DISEASE RELATIONSHIP: No evidence for a causal role for [GENE] in
[INHERITANCE & DISEASE] has been reported. Although this gene-disease relationship is supported by (relevant experimental evidence e.g. animal models,
expression studies, in vitro functional assays, etc.), no reports have directly implicated the gene in humans.
- DISPUTED: In summary, the evidence supporting the relationship between [GENE] and
[INHERITANCE & DISEASE] has been disputed and no valid evidence remains to support the claim. More evidence is needed to either support or entirely refute the role
[GENE] plays in this disease.
- REFUTED: In summary, the evidence supporting the relationship between [GENE] and
[INHERITANCE & DISEASE] has been refuted and no valid evidence remains to support the claim. [New evidence would be needed to support this claim.]
To maintain the provenance of all curations, please end with the following statement:
This classification was approved by the ClinGen [NAME OF GENE CURATION EXPERT
PANEL] on the meeting date [DATE] (SOP Version [#]).
Recuration
### For recurations, please end with the following statement:




This gene-disease pair was originally evaluated by the [GCEP name] on [original date]. It was reevaluated on [new date]. As a result of this reevaluation, the classification [did/did not change].
[If change, describe the original classification and the new classification. Include a brief description of why it changed - new literature, new SOP, etc.]
Example: This gene-disease pair was originally evaluated by the Aminoacidopathy GCEP on
June 29, 2020. It was re-evaluated on October 14, 2022. As a result of this reevaluation, the classification increased from Moderate to Definitive with the addition of a new mouse model
(PMIDs: 35135854).
[If no change, include a clear statement that either no new information is contributing to the classification OR that new evidence was evaluated but was not enough to change the classification.]
Example 1 (no new information): This classification was originally approved by the ClinGen
Hypertrophic Cardiomyopathy Gene Curation Expert Panel on August 1, 2017. This gene-
disease relationship was re-evaluated on September 14, 2022, by the Hereditary
Cardiovascular Disorders GCEP. As a result of this re-evaluation, the classification did not change.
Example 2 (new information, but not enough to change classification): This gene-disease pair was originally evaluated by the Hearing Loss GCEP on April 4, 2017. It was reevaluated on
November 17, 2021 (SOP v 8). Although a new animal model was published (PMID: 31628468),
the classification did not change.
In the event that a small change was made, such as a typo fix, include the following text:
This classification was approved by the [GCEP Name] Gene Curation Expert Panel on [original date]. As of [new date], this record underwent administrative updates [explain updates]. No new evidence has been reviewed or added.
Example: This classification was approved by the Intellectual Disability and Autism Gene
Curation Expert Panel on April 27, 2018. As of June 2022, this record underwent administrative updates to include an evidence summary text and update scoring to be consistent with SOP
Version 9. No new evidence has been reviewed or added.



### Appendix:

Evidence Summary examples (with hyperlinks)
For evidence summary examples from actual gene-disease pairs curated by approved ClinGen
GCEPs, please click on the hyperlinks below.
### Cardinal Gene-Disease Relationships:

These are well known, definitive gene-disease relationships. Often these gene-disease relationships have historical value, and an excess of genetic and/or experimental evidence.
1. NF2- Neurofibromatosis Type 2- autosomal dominant
2. FBN1- Marfan Syndrome- autosomal dominant
3. GBA- Gaucher disease - autosomal recessive
### Lumped Curations:

These examples include gene-disease relationships in which several asserted phenotypes and/or disease entities have been lumped into a broader entity. These may have included lumping eponymously named entities, phenotypic nomenclature, gene nomenclature,
biochemical nomenclature, or a combination of multiple types. Note, the indication of the entities that are lumped.
1. SCN2A- Complex Neurodevelopmental Disorder- autosomal dominant
2. CHD7- CHARGE syndrome- autosomal dominant
### Split curations:

These examples include gene-disease relationships in which one or more asserted disease entities and/or phenotypes have been split based on criteria from the ClinGen Lumping and
Splitting guidelines. Note, the indication of the entities that are excluded (split) from the present curation.
1. RET- Multiple endocrine neoplasia type 2B- autosomal dominant
2. MEF2C- Complex Neurodevelopmental Disorder- autosomal dominant
3. ACTG1- Baraitser-winter syndrome 2- autosomal dominant
### Recurations:

These examples include gene-disease relationships that have been re-evaluated. Note that these summaries may change with time dependent on the last approved classification.
1. AASS- Hyperlisinemia- autosomal recessive a. Classification changed
2. KLF10- Hypertrophic cardiomyopathy- autosomal dominant a. Classification did not change
3. TCAP- Hypertrophic cardiomyopathy- autosomal dominant a. Classification changed to disputed. Note the “reason for change” rationale

---

**Original File:** evidence_summary_v5_1.md
**Enhanced:** Evidence Summary Template v5.1
